doc/ProfilesJSONDocumentation.md

```diff
@@ -0,0 +1,57 @@
+| `defaultProfile` | _Required_ | String | Sets the default profile. Opens by typing <kbd>Ctrl</kbd> + <kbd>T</kbd> or by clicking the '+' icon. The guid of the desired default profile is used as the value. |
+| `initialCols` | _Required_ | Integer | The number of columns displayed in the window upon first load. |
+| `initialRows` | _Required_ | Integer | The number of rows displayed in the window upon first load. |
+| `requestedTheme` | _Required_ | String | Sets the theme of the application. Possible values: `"light"`, `"dark"`, `"system"` |
+| `showTerminalTitleInTitlebar` | _Required_ | Boolean | When set to `true`, titlebar displays the title of the selected tab. When set to `false`, titlebar displays "Windows Terminal". |
+| `experimental_showTabsInTitlebar` | Optional | Boolean | When set to `true`, the tabs are moved into the titlebar and the titlebar disappears. When set to `false`, the titlebar sits above the tabs. |
+| **profiles** | | **Array[Object]** | **Properties listed below are specific to each unique profile.** |
+| `acrylicOpacity` | _Required_ | Number | When `useAcrylic` is set to `true`, it sets the transparency of the window for the profile. Accepts values from 0-1. |
+| `background` | _Required_ | String | Sets the background color of the profile. Overrides `background` set in color scheme if `colorscheme` is set. Uses hex color format: `"#rrggbb"`. |
+| `closeOnExit` | _Required_ | Boolean | When set to `true`, the selected tab closes when `exit` is typed. When set to `false`, the tab will remain open when `exit` is typed. |
+| `colorscheme` | _Required_ | String | Name of the terminal color scheme to use. Color schemes are defined under `schemes`. |
+| `commandline` | _Required_ | String | Executable used in the profile. |
+| `cursorColor` | _Required_ | String | Sets the cursor color for the profile. Uses hex color format: `"#rrggbb"`. |
+| `cursorShape` | _Required_ | String | Sets the cursor shape for the profile. Possible values: `"vintage"`, `"bar"`, `"underscore"`, `"filledBox"`, `"emptyBox"` |
+| `fontFace` | _Required_ | String | Name of the font face used in the profile. |
+| `fontSize` | _Required_ | Integer | Sets the font size. |
+| `guid` | _Required_ | String | Unique identifier of the profile. Written in registry format: `"{00000000-0000-0000-0000-000000000000}"`. |
+| `historySize` | _Required_ | Integer | The number of lines above the ones displayed in the window you can scroll back to. |
+| `name` | _Required_ | String | Name of the profile. Displays in the dropdown menu. |
+| `padding` | _Required_ | String | Sets the padding around the text within the window. Can have three different formats: `"#"` sets the same padding for all sides, `"#, #"` sets the same padding for top-bottom and right-left, and `"#, #, #, #"` sets the padding individually for top, right, bottom, and left. |

```

Comment:
I am pretty sure that the order is "top, left, bottom, right". I've never seen a rectangle specified with right before left...

Reply:
Same goes for top-bottom and left-right...

Reply:
I used this for reference: https://www.w3schools.com/cssref/pr_padding.asp

Reply:
OK, but this isn't CSS.

This is the code calling for the thicknesses to be set up (for padding):
https://github.com/microsoft/terminal/blob/master/src/cascadia/TerminalControl/TermControl.cpp#L1338

And here's the function specification:
https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.thicknesshelper.fromlengths#Windows_UI_Xaml_ThicknessHelper_FromLengths_System_Double_System_Double_System_Double_System_Double_

Which actually means we're both wrong. It's Left, Top, Right, Bottom.